src/config/config.ts

```diff
@@ -19,8 +17,6 @@ export class Configuration {
   defaultVolumeDecimal = 2;
   defaultPercentageDecimal = 2;
```

Comment:
The following config entries should IMO be unused and can be removed:
- githubToken = process.env.GH_TOKEN;
- defaultLanguage = 'en';
- defaultCountry = 'DE';
- defaultCurrency = 'EUR';
- defaultVolumeDecimal = 2;
- defaultPercentageDecimal = 2;